test/AspNetCoreModule.Test/FunctionalTestHelper.cs

```diff
@@ -35,7 +35,8 @@ public bool IsMet
         {
             get
             {
-                if (TestFlags.Enabled(TestFlags.SkipTest))
+                if (_attributeValue == TestFlags.SkipTest
```

Comment:
In this condition, test are always got skipped, which is a breaking change to our test pass. So, please update this in order to make some way not to skip whole tests.